hr_attendance_report_theoretical_time/views/hr_employee_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <act_window
+        id="recompute_employee_theoretical_attendance"
+        name="Recompute Theoretical Attendance"
+        binding_model="hr.employee"
+        res_model="recompute.theoretical.attendance"
+        view_mode="form"
+        context="{'default_employee_ids': active_ids, 'employee_ids': active_ids}"
+        target="new"
+        groups="hr_attendance.group_hr_attendance_manager"
+    />
```

Comment:
Runbot is red because of this definition:

```
2021-07-02 09:08:08,833 194 WARNING openerp_test py.warnings:
/home/odoo/odoo-14.0/odoo/tools/convert.py:336: 
DeprecationWarning: The <act_window> tag is deprecated,
use a <record> for 'recompute_employee_theoretical_attendance'.
warnings.warn(f"The <act_window> tag is deprecated,
use a <record> for {xml_id!r}.", DeprecationWarning)
```
Can you check it please.